tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -25,6 +25,7 @@ limitations under the License.
 #include <limits>
 #include <memory>
 #include <type_traits>
+#include <vector>
```

Comment:
Why is this new header needed? I don't see any usage of std::vector in the changed code.

Reply:
As per discussion with @petewarden, this include was not needed as some cpplint include warnings can be ignored.  A new commit will be made to fix this.